src/layers/StaticImageLayer.js

```diff
@@ -66,7 +78,6 @@ export default class StaticImageLayer extends CompositeLayer {
     const { data } = this.state;
     return new XRLayer({
       channelData: data,
-      data: null,
       bounds,
       // Going forward, we should have more intricate indexing so we can
```

Comment:
I think we can remove this comment now?  This PR + the colormap one from the weekend seems to have more or less solved the issue this comment references.